### PR TITLE
feat(941220): add regexp-assembly for deconstructed regex

### LINKED ISSUE #2580: Dissasemble regexes in rules 941210 and 941220
### Motivation

All regexes should be generated using regexp-assembly.

### Proposed solution

Dissasemble the regexp in rules 941210 and 941220.

### PR DESCRIPTION
**Pull request by fzipi**
Complements #3262.

Fixes #2580.

We can probably create an include file with the "script" part, but I don't know if it makes too much sense.

### CODE REVIEW

**Comment by RedXanadu** [comment]
The documentation here is great :smile: Hopefully we can bring all the regex assemble files up to this level.

File: regex-assembly/941220.ra [hunk 1]
```diff
@@ -0,0 +1,165 @@
+##! Please refer to the documentation at
+##! https://coreruleset.org/docs/development/regex_assembly/.
+
+##!+ i
```
Comment (theseion) [inline regex-assembly/941220.ra:4]: Move to after the rule comment.

File: regex-assembly/941220.ra [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+##! This rule tries to match all the possible ways to write 'vbscript' using
+##! unicode characters, html entities, and javascript escape sequences.
+
+##! Matched order is sequential: we first match 'v', then 'b', then 's', etc.
+
+##!> define html_entity_prefix &#x?0*
+##!> define whitespaces \t|\n|\r
```
Comment (theseion) [inline regex-assembly/941220.ra:12]:
```suggestion
##!> define whitespaces [\t\n\r]
```

**Review by theseion — CHANGES_REQUESTED** [review]
I'll wait for you to make the changes to the other rule. The comments there apply here as well.

**Comment by theseion** [comment]
I don't think moving the `script` part (or individual characters) to an include file is necessary. We can discuss it when we need it a third time.

**Comment by fzipi** [comment]
Updated similar to the other rule.

**Review by theseion — APPROVED** [review]

**Pull request merged**